template.go

```diff
@@ -132,6 +132,11 @@ func (r {{$typename}}) MarshalCSV() (string, error) {
     return r.getString()
 }
 
+// UnmarshalCSV is generated so {{$typename}} satisfies csv.Unmarshaler.
+func (r {{$typename}}) UnmarshalCSV(s string) (error) {
```

Comment:
I think this needs to be a pointer receiver

Reply:
thanks. not having tests to rely on is scary 38a278e

Reply:
that's what code review is for :)